### DISCUSSION TITLE
Where can I see the report on how the performance of the MinIO cluster changes with the number of nodes and disks?

### DISCUSSION THREAD

**Discussion by adababys** [minio/minio#19471] [Q&A]
Where can I see the report on how the performance of the MinIO cluster changes with the number of nodes and disks?

**Comment by klauspost** [comment]
@shaolila Generally we scale very close to linear for the same availability requirements.

For detailed performance queries email `sales@min.io` with your requirements and we can assist you in choosing an appropriate setup.
